user-operator/src/main/java/io/strimzi/operator/user/UserOperator.java

```diff
@@ -30,7 +30,7 @@
 
     public static final String STRIMZI_CLUSTER_OPERATOR_DOMAIN = "user.operator.strimzi.io";
 
-    private static final int HEALTH_SERVER_PORT = 8080;
+    private static final int HEALTH_SERVER_PORT = 8081;
```

Comment:
Have you updated the port also in the User Operator deployment?

Reply:
Yeah :-)